streamlit_web.py

```diff
@@ -657,23 +662,73 @@ def interactive():
     if ner_cb:
         doc = az.get_nlp(input_text)
         named_entities = az.named_entity_recognization(input_text)
-        if len(named_entities) > 0:
-            html = spacy.displacy.render(doc, style="ent")
-            # Newlines seem to mess with the rendering
-            html = html.replace("\n", " ")
-            HTML_WRAPPER = """<div style="overflow-x: auto; border: 1px solid \
-        #e6e9ef; border-radius: 0.25rem; padding: 1rem;
-        margin-bottom: 2.5rem">\
-        {}</div>"""
-            st.write(HTML_WRAPPER.format(html), unsafe_allow_html=True)
-        else:
-            st.info("No named entity recognized")
+        displacy_renderer(named_entities)
     if sentiment_cb:
         sentiments = TextBlob(az.lemmatized_text(input_text))
         st.write(sentiments.sentiment)
     if summary_cb:
         summaries = sz.summarize_text(input_text)
         st.write(summaries)
 
+
+def entities():
+    """Page to display entity analysis"""
+    st.write("Entity analysis inspects the given text for known entities \
+    and returns information about those entities. It is a way to extract \
+    information that seeks to locate and classify named entities in text \
+    into pre-defined categories such as the names of persons, organizations, \
+    locations, expressions of times, quantities, monetary values, and percentages.")
+
+    # make a copy of the main dataframe
+    input_df = main_df.copy(deep=True)
+    input_df = input_df[input_df[assign_id].isin(assignments)]
+
+    # makes a drop down list to select users classified by assignments
+    for assignment in input_df[assign_id].unique():
+        st.write("")
```

Comment:
Is this line needed since it's not writing anything?

Reply:
Yes, this was on purpose. I put that there to get an empty line because the assignment header was too close to the student list. It's only used for styling purposes.